vue/src/api/api.js

```diff
@@ -1,41 +1,37 @@
 import axios from 'axios';
 
 const api = axios.create({
-    baseURL: '/api/v2/',
-    timeout: 10000,
+    baseURL: 'http://localhost:8081/api/v2/',
```

Comment:
Yep, I moved it to that since I was testing this via caddy and nginx as well as `/vue/` in Medusa.